### PR TITLE
expose form values to parent component

### PR DESCRIPTION
**Pull request by zackify**
@audiolion thoughts? Should be good to go :)

### CODE REVIEW

File: tests/expose-values.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+//Test things dealing with the initialValues prop
```
Comment (audiolion) [inline tests/expose-values.js]: ~~initialValues~~ => values

File: tests/expose-values.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import React from 'react';
+import { spy } from 'sinon';
+import Form from '../src/form';
+import { shallow } from 'enzyme';
+
+const Input = ({ error, ...props }) =>
+  error ? <p className="error">{error}</p> : <input {...props} />;
+
+test('Form passes values to onValues', () => {
+  const onValues = spy();
```
Comment (audiolion) [inline tests/expose-values.js]: jest has [its own spy](https://facebook.github.io/jest/docs/jest-object.html#jestspyonobject-methodname), do you just prefer `sinon`'s implementation?

File: README.md [hunk 3]
```diff
@@ -66,12 +66,18 @@ errorMessages={{
   'min.password': 'Custom min password required error message.'
 }}
 ```
+**onValues**
 
-**initialValues**
+exposes the values on change, you must manage the form state by passing in values if using this.
```
Comment (audiolion) [inline README.md]: I might say `passing in state to the ``values`` prop if using this.` to make it more obvious.

File: tests/expose-values.js (hunk 2 above)
Comment (zackify) [inline tests/expose-values.js]: ooo i used it back before i used jest and didnt bother to look and see. maybe ill make a pr where i remove it soon. You're welcome to if you want as well.

**Pull request merged**